Kickstarter-iOS/Views/Controllers/SortPagerViewController.swift

```diff
@@ -138,7 +138,7 @@ internal final class SortPagerViewController: UIViewController {
 
   private func createSortButtons(sorts: [DiscoveryParams.Sort]) {
     self.sortsStackView
-      |> UIStackView.lens.arrangedSubviews .~ sorts.enumerate().map { idx, sort in
+      |> UIStackView.lens.arrangedSubviews .~ sorts.enumerate().map { idx, _ in
```

Comment:
it should! but in fact it's done in #32 cause that file is in that target instead of the library target.

good 👀 !